### ISSUE TITLE
Link is not working for pretrained model

### ISSUE THREAD

**Issue by Datastar07** [lalaique/deep-floor-plan-recognition#2]
i am trying the download pretrained weight of the model but it's not worked.

also trying microsoft link but it's also not worked.

**Comment by sultonouv** [comment]
The link is broken. Update the link to the weights please!

**Comment by MikeMpapa** [comment]
yes please - all provided links both in README and [here](https://github.com/whchien/deep-floor-plan-recognition/issues/1) seem to be broken

**Comment by vladBaluta** [comment]
can you please provide me access to the model? I did not find in these conversations any link that still works. Thank you!
